content/_partials/toptoolbar.html.haml

```diff
@@ -13,71 +13,67 @@
 
         %li.nav-item.dropdown
           .nav-link.dropdown-toggle{"aria-expanded" => "false", "aria-haspopup" => "true", "data-toggle" => "dropdown", :role => "button"}
-            Documentation
+            Docs
```

Comment:
Personal preference, or why this change?

Reply:
Reducing width.  I can revert.